<h1>Evolutionary Genetics of Humans and Other Primates: A Journal Club</h1>

<h2>Introduction</h2>

<p>Welcome to eghop, a journal club on the evolutionary genetics of
humans and other primates. We meet weekly by Zoom. During the Spring
semester of 2021, we'll meet Tuesdays at 3PM
<a href="https://en.wikipedia.org/wiki/Mountain_Time_Zone">Mountain time</a>,
which is UTC-7 until March 14, then UTC-6.</p>

<p>Each week, we discuss a recent publication or preprint on the
evolutionary genetics of humans or other primates. Everyone is
welcome, from anywhere in the world. You need not be affiliated with
our University, and there is no fee. Undergraduates are welcome.  The
only prerequisite is an interest in the subject. The journal club is
administered by Alan Rogers and Tim Webster, of the University of
Utah.</p>

<p>Participants are expected to treat each other with respect. Anyone who
fails to do so will be blocked.</p>

<p>If you'd like to participate, send email to Alan Rogers
(<a href="mailto:rogers@anthro.utah.edu">rogers@anthro.utah.edu</a>). Unless I
know you already, tell me about yourself. I'll send you the Zoom
meeting id and password.</p>

<h2>Suggested readings</h2>

<p>Click <a href="suggested.html">here</a> for the list of readings that have been
suggested for future meetings of the journal club. Use
<a href="https://docs.google.com/forms/d/e/1FAIpQLSeC-CnDFiBVV5Zw_SR-CrBe5zYSijwkM1aiIqKeDjeQX2eRQQ/viewform?usp=sf_link">this link</a> to suggest an article or preprint yourself. We
prefer articles that present new findings--either empirical or
theoretical. Review papers and methodological papers are less likely
to provoke interesting discussions. We'll conduct surveys from time to
time to vote on the suggested readings.</p>

<h2>History</h2>

<p>Click <a href="past.html">here</a> for a history of past journal club meetings.</p>

<h2>Schedule</h2>

<ul>
<li><p>Jan 26 <a href="https://www.cell.com/cell/fulltext/S0092-8674(19)30218-1">Jacobs et al 2019</a> Multiple deeply divergent
Denisovan ancestries in Papuans. Presenter: Alan Rogers.</p>

<ul>
<li>"Modern Papuans carry hundreds of gene variants from two deeply
divergent Denisovan lineages that separated over 350 thousand
years ago. Spatial and temporal structure among these lineages
suggest that introgression from one of these Denisovan groups
predominantly took place east of the Wallace line and continued
until near the end of the Pleistocene. A third Denisovan lineage
occurs in modern East Asians. This regional mosaic suggests
considerable complexity in archaic contact, with modern humans
interbreeding with multiple Denisovan groups that were
geographically isolated from each other over deep evolutionary
time."</li>
</ul></li>
<li><p>Feb 02 <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1008552">Wang et al. 2020</a> Tracking human population structure
through time from whole genome sequences.</p>

<ul>
<li>These authors introduce a new method for inferring the deep
history of population size and admixture. Using this method, they
"detect traces of extremely deep ancestry between some African
populations, with around 1% of ancestry dating to divergences older
than a million years ago."</li>
</ul></li>
<li><p>Feb 09 <a href="https://www.mdpi.com/2073-4425/11/12/1510">Vizzari et al 2020</a> A revised model of
anatomically modern human expansions out of Africa through a
machine learning approximate Bayesian computation approach.</p>

<ul>
<li>The authors argue that modern humans emigrated out of Africa in
two waves, first at 74 kya and later at 46 kya.</li>
</ul></li>
<li><p>Feb 16 <a href="https://www.biorxiv.org/content/10.1101/2021.01.22.427608v1">Andirko et al. 2021</a> Fine-grained temporal
mapping of derived high-frequency variants supports the mosaic
nature of the evolution of Homo sapiens. </p>

<ul>
<li>These authors summarize published data on the time of origin of
high-frequency alleles in several categories: those under positive
selection, those introgressed from archaics, and regulatory genes
active in the brain. Several of these age distributions are
bimodal, with one mode at 300-500 kya.</li>
</ul></li>
<li><p>Feb 23 <a href="https://www.pnas.org/content/118/7/e2010632118">Orkin et al 2020</a> <a href="https://www.biorxiv.org/content/10.1101/366112v6.full">bioRxiv</a> The
evolution of ecological flexibility, large brains, and long lives:
capuchin monkey genomics revealed with fecalFACS.</p>

<ul>
<li>"We compared genomes of capuchin populations from tropical dry
forests and lowland rainforests and identified population
divergence in genes involved in water balance, kidney function,
and metabolism. Through a comparative genomics approach spanning a
wide diversity of mammals, we identified genes under positive
selection associated with longevity and brain development."</li>
</ul></li>
<li><p>Mar 02</p></li>
<li><p>Mar 09</p></li>
<li><p>Mar 16</p></li>
<li><p>Mar 23</p></li>
<li><p>Mar 30</p></li>
<li><p>Apr 06</p></li>
<li><p>Apr 13</p></li>
<li><p>Apr 20</p></li>
<li><p>Apr 27</p></li>
</ul>
