<p><head>
<title>Summer Journal Club on Anthropological Genetics</title>
</head></p>

<h1>Evolutionary genetics of humans and other primates: a journal club</h1>

<h2>Introduction</h2>

<p>Welcome to eghop, a journal club on the evolutionary genetics of
humans and other primates. We meet by Zoom <em>every Monday at 1 PM
<a href="https://en.wikipedia.org/wiki/Mountain_Time_Zone">Mountain time</a></em>.</p>

<p>Each week, we'll discuss a recent publication or preprint on the
evolutionary genetics of humans or other primates. Everyone is
welcome, from anywhere in the world. You need not be affiliated with
our University, and there is no fee. Undergraduates are welcome.  The
only prerequisite is an interest in the subject. The journal club is
administered by Alan Rogers and Tim Webster, of the University of
Utah.</p>

<p>Participants are expected to treat each other with respect. Anyone who
fails to do this will be blocked.</p>

<p>If you'd like to participate, send email to Alan Rogers
(rogers@anthro.utah.edu). Unless we know you already, tell us about
yourself. We'll send you the Zoom meeting id and password.</p>

<h2>Suggested readings</h2>

<p>Please use <a href="https://docs.google.com/forms/d/e/1FAIpQLSeC-CnDFiBVV5Zw_SR-CrBe5zYSijwkM1aiIqKeDjeQX2eRQQ/viewform?usp=sf_link">this link</a> to suggest articles for discussion. We
prefer articles that present new findings--either empirical or
theoretical. Review papers and methodological papers are less likely
to provoke interesting discussions. We'll conduct surveys from time to
time to vote on the suggested readings.</p>

<ol>
<li><p><a href="https://www.nature.com/articles/s41559-020-1261-z">Rinker et al. 2020</a> Neanderthal introgression
reintroduced functional ancestral alleles lost in Eurasian
populations.</p>

<ul>
<li>"Our study reveals that Neanderthal introgression reintroduced
thousands of lost ancestral variants with gene regulatory activity
and that these RAs [reintroduced alleles] were more tolerated than
NDAs [Neanderthal-derived allelees]. Thus, RAs and their distinct
evolutionary histories must be considered when evaluating the
effects of introgression."</li>
</ul></li>
<li><p><a href="https://www.nature.com/articles/s41586-020-2487-2">Ioannidis et al. 2020</a> Native American gene flow
into Polynesia predating Easter Island settlement.</p>

<ul>
<li>"Here we analyse genome-wide variation in individuals from
islands across Polynesia for signs of Native American admixture,
analysing 807 individuals from 17 island populations and 15
Pacific coast Native American groups. We find conclusive evidence
for prehistoric contact of Polynesian individuals with Native
American individuals (around ad 1200) contemporaneous with the
settlement of remote Oceania. Our analyses suggest
strongly that a single contact event occurred in eastern
Polynesia, before the settlement of Rapa Nui, between Polynesian
individuals and a Native American group most closely related to
the indigenous inhabitants of present-day Colombia." </li>
</ul></li>
<li><p><a href="https://www.nature.com/articles/s41562-020-0897-7">Betti et al. 2020</a> Climate shaped how Neolithic
farmers and European hunter-gatherers interacted after a major
slowdown from 6,100 BCE to 4,500 BCE.</p>

<ul>
<li>"We identify four axes of [European Neolithic] expansion and
observe a slowdown along three axes when crossing the same
climatic threshold. This threshold reflects the quality of the
growing season, suggesting that Near Eastern crops might have
struggled under more challenging climatic conditions. This same
threshold also predicts the mixing of farmers and hunter-gatherers
as estimated from ancient DNA, suggesting that unreliable yields
in these regions might have favoured the contact between the two
groups."</li>
</ul></li>
<li><p><a href="https://www.biorxiv.org/content/10.1101/2020.07.03.186296v1">Zeberg and Paabo. 2020</a> The major genetic risk
factor for severe COVID-19 is inherited from Neandertals.</p>

<ul>
<li>"A recent genetic association study (Ellinghaus et al. 2020)
identified a gene cluster on chromosome 3 as a risk locus for
respiratory failure in SARS-CoV-2. Recent data comprising 3,199
hospitalized COVID-19 patients and controls reproduce this and
find that it is the major genetic risk factor for severe
SARS-CoV-2 infection and hospitalization (COVID-19 Host Genetics
Initiative). Here, we show that the risk is conferred by a genomic
segment of ~50 kb that is inherited from Neandertals and occurs at
a frequency of ~30% in south Asia and ~8% in Europe."</li>
</ul></li>
<li><p><a href="https://academic.oup.com/mbe/article/doi/10.1093/molbev/msaa140/5874945?fbclid=IwAR3FN-AN8t7pFfz3OMNrC_22JA1bzrLPqSQswSoWqMjhZz3T_P3mwANGWTw#.XxmB88-hF6o.facebook">Schlebusch et al. 2020</a> Khoe-San genomes reveal
unique variation and confirm the deepest population divergence in
<em>Homo sapiens</em>.</p>

<ul>
<li>Sequenced 25 full genomes from Koe-San populations in southern
Africa. Find that Koe-San have more genetic diversity than any
other population and that 25% of variants are unique to Koe-San.
Evidence for selection at genes involved in immunity, the brain,
sperm, diet, and muscle. Evidence of a decline in population size
100,000 years ago.</li>
</ul></li>
<li><p><a href="https://www.biorxiv.org/content/10.1101/851188v1">Immel et al. 2019.</a> Neolithic genomes reveal a
distinct ancient HLA allele pool and population transformation in
Europe.</p>

<ul>
<li>Studied 42 late-Neolithic genomes from the Wartenberg culture of
Europe. A high percentage--40%--of their ancestry was from hunter
gatherers, and their HLA alleles were unlike those of modern
Europeans. The authors argue that their immune systems were
specialized for defense against viruses.</li>
</ul></li>
<li><p><a href="https://www.biorxiv.org/content/10.1101/427583v2?%3Fcollection=">Zen, Huber, Davies, and Lohmuller, 2020</a> Greater
strength of selection and higher proportion of beneficial amino
acid changing mutations in humans compared to mice and Drosophila
melanogaster.</p>

<ul>
<li>"We use improved modeling of weakly deleterious mutations and the
demographic history of the outgroup species and ancestral
population and estimate that at least 20% of nonsynonymous
substitutions between humans and an outgroup species were fixed by
positive selection. This estimate is much higher than previous
estimates, which did not correct for the sizes of the outgroup
species and ancestral population."</li>
</ul></li>
<li><p><a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msaa119/5841671">Zeberg, et al. 2020.</a> The Neandertal Progesterone
Receptor.</p>

<ul>
<li>"The gene encoding the progesterone receptor (PGR) carries
introgressed Neandertal haplotypes...[, which] have reached nearly
20% frequency in non-Africans and have been associated with
preterm birth...  Present-day carriers of the Neandertal
haplotypes express higher levels of the receptor. In a cohort of
present-day Britons, these carriers have more siblings, fewer
miscarriages, and less bleeding during early pregnancy suggesting
that the Neandertal progesterone receptor alleles promote
fertility. This may explain their high frequency in modern human
populations."</li>
</ul></li>
<li><p><a href="https://academic.oup.com/mbe/article/34/10/2704/3988100">Xu et al. 2017.</a> Archaic hominin introgression in Africa
contributes to functional salivary MUC7 genetic variation.</p>

<ul>
<li>Unlike previous authors, these authors find no association between
the MUC7 salivary protein and asthma. On the other hand, this
locus is associated with the oral microbiome. Furthermore, one
haplotype within this locus seems to have originated in an unknown
but deeply-divergent (superarchaic?) population, which
introgressed into the ancestors of modern Africans.</li>
</ul></li>
<li><p><a href="https://www.biorxiv.org/content/10.1101/2020.04.16.045450v1">Hernandez, Shenk, and Perry. 2020</a> Factors
influencing taxonomic unevenness in scientific research: A
mixed-methods case study of non-human primate genomic sequence data
generation.</p>

<ul>
<li>How do scientists decide which species to focus on in
DNA-sequencing studies? This study shows that the most-studied
species are those for which there is a large prior literature and
that there is a bias in favor of species closely-related to our
own.</li>
</ul></li>
<li><p><a href="https://advances.sciencemag.org/content/6/24/eaaz0183.abstract">Wang et al. 2020.</a> Ancient genomes reveal complex
patterns of population movement, interaction, and replacement in
sub-Saharan Africa.</p>

<ul>
<li>"Here, we report genome-wide data from 20 ancient sub-Saharan
African individuals, including the first reported ancient DNA from
the DRC, Uganda, and Botswana. These data demonstrate the
contraction of diverse, once contiguous hunter-gatherer
populations, and suggest the resistance to interaction with
incoming pastoralists of delayed-return foragers in aquatic
environments. ... In Botswana, we show that Bantu ancestry
post-dates admixture between pastoralists and foragers, suggesting
an earlier spread of pastoralism than farming to southern Africa."</li>
</ul></li>
<li><p><a href="https://www.nature.com/articles/s41467-018-04204-w">Hicks et al. 2018.</a> Gut microbiomes of wild great apes
fluctuate seasonally in response to diet.</p>

<ul>
<li>"Here we use 16S rRNA sequencing to characterize the microbiota of
wild western lowland gorillas and sympatric central chimpanzees
and demonstrate compositional divergence between the microbiotas
of gorillas, chimpanzees, Old World monkeys, and modern humans. We
show that gorilla and chimpanzee microbiomes fluctuate with
seasonal rainfall patterns and frugivory."</li>
</ul></li>
<li><p><a href="https://www.biorxiv.org/content/10.1101/2020.06.25.170308v1.full?%3Fcollection=">Chakraborty et al. 2020.</a> Whole genomes
reveal severe bottleneck among Asian hunter-gatherers following the
invention of agriculture.</p>

<ul>
<li>"Using genomic data, we show that for the majority of Asian
hunter-gatherers, population-sizes drastically decreased following
the invention of agriculture. We argue that a combination of
displacement, enslavement and disease resulted in the decimation
of hunter-gatherer societies."</li>
</ul></li>
<li><p><a href="https://www.nature.com/articles/s41598-020-66867-0">Cawthon et al. 2020.</a> Germline mutation rates in
young adults predict longevity and reproductive lifespan.</p>

<ul>
<li>"The paper is the first report showing that differences in the
levels of mutation accumulation in young healthy adults predict
remaining lifespan in both sexes, and remaining reproductive span
in women. It is also the first report showing that mutation
accumulation accelerates in females after menarche, and it
presents data suggesting that mutation accumulation accelerates in
males after puberty as well. These results support two
long-standing theories about the nature of aging: 1) aging is due,
at least in part, to mutation accumulation in adults contributing
to cell and tissue dysfunction, onset of chronic diseases, and
ultimately death; and 2) aging begins in adolescence, possibly
triggered by puberty, consistent with the evolutionary biology
theory that the force of natural selection to maintain robust
health should begin to decline once the reproductive stage of life
is attained."</li>
</ul></li>
<li><p><a href="https://science.sciencemag.org/content/368/6498/1495.abstract">Sinding et al. 2020.</a> Arctic-adapted dogs emerged at
the Pleistocene-Holocene transition.</p>

<ul>
<li>"This analysis indicates that sled dogs represent an ancient
lineage going back at least 9500 years and that wolves bred with
the ancestors of sled dogs and precontact American dogs. However,
gene flow between sled dogs and wolves likely stopped before âˆ¼9500
years ago."</li>
</ul></li>
</ol>

<h2>Schedule</h2>

<ul>
<li><p>May 25 <a href="https://www.nature.com/articles/s41586-020-2225-9?fbclid=IwAR2KASFusrFuvKtuHmR3gBfFX5NjzMKeJ1azAVNWQY0epnMDY8D5yNRGsMk">Skov et al 2020</a> The nature of Neanderthal
introgression revealed by 27,566 Icelandic genomes. Presenter: Alan
Rogers. </p></li>
<li><p>Jun 01 <a href="https://www.cell.com/trends/genetics/fulltext/S0168-9525(20)30070-6">Rees, Castellano, and Andres. 2020.</a> The
genomics of local adaptation. Presenter: Jasmin Rees.</p></li>
<li><p>Jun 08 <a href="https://www.cell.com/cell/fulltext/S0092-8674(20)30502-X?fbclid=IwAR3dddrez6WPCZctlDiIVsfagqAXWuA9rVACmlXDhfSzcZIvox8N66WmOew">Yu et al. 2020.</a> Paleolithic to Bronze Age Siberians
Reveal Connections with First Americans and across
Eurasia. Presenter: Alan Rogers</p>

<ul>
<li><p>Nineteen Upper Paleolithic to Early Bronze Age genomes from the
Lake Baikal region of Siberia. One Upper Paleolithic genome shares
the admixed ancestry that gave rise to all non-Arctic Native
Americans.</p></li>
<li><p>Populations of the Bronze Age in this region formed from those of
the Neolithic by prolonged admixture.</p></li>
<li><p>Long-range human and Y. pestis mobility across Eurasia during the
Early Bronze Age.</p></li>
</ul></li>
<li><p>Jun 15 <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1807-z">Amato et al. 2019</a> Convergence of human and Old World
monkey gut microbiomes demonstrates the importance of human ecology
over phylogeny. Presenter: Katie Amato.</p>

<ul>
<li>"Human gut microbiome composition and functional potential are
more similar to those of cercopithecines, a subfamily of Old World
monkey, particularly baboons, than to those of African apes." The
microbiomes of humans are also more variable than those of other
primates.</li>
</ul></li>
<li><p>Jun 22 <a href="https://doi.org/10.1371/journal.pgen.1008293">Musharoff, et al. 2019.</a> The inference of
sex-biased human demography from whole-genome data. Presenter:
Shaila Musharoff. </p>

<ul>
<li>X chromosomes and autosomes have different levels of genetic
diversity because of their different population
sizes. Furthermore, they respond differently to changes in
population size, even in the absence of differences in male and
female effective population sizes. These authors present a new
method that simultaneously estimates changes in population size
and in the ratio of male to female effective sizes. Finally, they
apply the new method to several human populations.</li>
</ul></li>
<li><p>Jun 29 <a href="https://advances.sciencemag.org/content/6/8/eaay5483">Rogers, Harris, and Achenbach. 2020.</a>
Neanderthal-Denisovan ancestors interbred with a distantly-related
hominin. Presenter: Alan Rogers</p>

<ul>
<li>"We show that early in the middle Pleistocene, long before the
expansion of modern humans into Eurasia, the 'neandersovan'
ancestors of Neanderthals and Denisovans undertook a very similar
expansion. In both cases, an African population expanded into
Eurasia, endured a narrow bottleneck of population size, interbred
with indigeneous Eurasians, largely replaced them, and split into
eastern and western sub-populations. In the earlier expansion,
neandersovans interbred with a 'superarchaic' population that had
been separate since about 2 mya and may represent the original
expansion of humans into Eurasia."</li>
</ul></li>
<li><p>Jul 06 Two readings: (1) <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0186237">Munung et al. 2017</a> Equity in
international health research collaborations in Africa: Perceptions
and expectations of African researchers, and (2)
<a href="https://advances.sciencemag.org/content/5/10/eaaw7238.abstract">Hoppe et al. 2019.</a> Topic choice contributes to the
lower rate of NIH awards to African-American/black
scientists. Presenter: Tina Lasisi and Tim Webster. </p></li>
<li><p>Jul 13 <a href="https://www.sciencedirect.com/science/article/pii/S0959437X1730182X?via%3Dihub">Henn, Steele, and Weaver. 2018.</a> Clarifying
distinct models of modern human origins in Africa. Presenter:
Brenna Henn.</p>

<ul>
<li>"We describe four possible models for the origins of Homo sapiens
in Africa based on published literature from paleoanthropology and
human genetics. We briefly outline expectations for data patterns
under each model, with a special focus on genetic data." </li>
</ul></li>
<li><p>Jul 20 <a href="https://www.biorxiv.org/content/10.1101/2020.03.13.990523v1">Peter. 2020</a> 100,000 years of gene flow between
Neandertals and Denisovans in the Altai mountains. Presenter:
Benjamin Peter.</p>

<ul>
<li>Studies gene flow between Neanderthals and Denisovans using a new
method that can cope with degraded, low-coverage, DNA. Finds
evidence of gene flow in both directions over a long period of
time. But only in the Altai region of Siberia. European
Neanderthals show no evidence of Denisovan admixture.</li>
</ul></li>
<li><p>Jul 27 <a href="https://www.biorxiv.org/content/10.1101/2020.06.01.127555v1">Cole et al. 2020.</a> Ancient admixture into
Africa from the ancestors of non-Africans. Presenter: Chris Cole. </p>

<ul>
<li>"Here we present SMCSMC, a Bayesian method for inference of
time-varying population sizes and directional migration rates
under the coalescent-with-recombination model, to study ancient
demographic events. We find evidence for substantial migration
from the ancestors of present-day Eurasians into African groups
between 40 and 70 thousand years ago, predating the divergence of
Eastern and Western Eurasian lineages. This event accounts for
previously unexplained genetic diversity in African populations,
and supports the existence of novel population substructure in the
Late Middle Paleolithic."</li>
</ul></li>
<li><p>Aug 03</p></li>
<li><p>Aug 10</p></li>
<li><p>Aug 17</p></li>
</ul>
