<h1>Past discussion topics</h1>

<h2>Summer 2020</h2>

<ul>
<li><p>May 25 <a href="https://www.nature.com/articles/s41586-020-2225-9?fbclid=IwAR2KASFusrFuvKtuHmR3gBfFX5NjzMKeJ1azAVNWQY0epnMDY8D5yNRGsMk">Skov et al 2020</a> The nature of Neanderthal
introgression revealed by 27,566 Icelandic genomes. Presenter: Alan
Rogers. </p></li>
<li><p>Jun 01 <a href="https://www.cell.com/trends/genetics/fulltext/S0168-9525(20)30070-6">Rees, Castellano, and Andres. 2020.</a> The
genomics of local adaptation. Presenter: Jasmin Rees.</p></li>
<li><p>Jun 08 <a href="https://www.cell.com/cell/fulltext/S0092-8674(20)30502-X?fbclid=IwAR3dddrez6WPCZctlDiIVsfagqAXWuA9rVACmlXDhfSzcZIvox8N66WmOew">Yu et al. 2020.</a> Paleolithic to Bronze Age Siberians
Reveal Connections with First Americans and across
Eurasia. Presenter: Alan Rogers</p>

<ul>
<li><p>Nineteen Upper Paleolithic to Early Bronze Age genomes from the
Lake Baikal region of Siberia. One Upper Paleolithic genome shares
the admixed ancestry that gave rise to all non-Arctic Native
Americans.</p></li>
<li><p>Populations of the Bronze Age in this region formed from those of
the Neolithic by prolonged admixture.</p></li>
<li><p>Long-range human and Y. pestis mobility across Eurasia during the
Early Bronze Age.</p></li>
</ul></li>
<li><p>Jun 15 <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1807-z">Amato et al. 2019</a> Convergence of human and Old World
monkey gut microbiomes demonstrates the importance of human ecology
over phylogeny. Presenter: Katie Amato.</p>

<ul>
<li>"Human gut microbiome composition and functional potential are
more similar to those of cercopithecines, a subfamily of Old World
monkey, particularly baboons, than to those of African apes." The
microbiomes of humans are also more variable than those of other
primates.</li>
</ul></li>
<li><p>Jun 22 <a href="https://doi.org/10.1371/journal.pgen.1008293">Musharoff, et al. 2019.</a> The inference of
sex-biased human demography from whole-genome data. Presenter:
Shaila Musharoff. </p>

<ul>
<li>X chromosomes and autosomes have different levels of genetic
diversity because of their different population
sizes. Furthermore, they respond differently to changes in
population size, even in the absence of differences in male and
female effective population sizes. These authors present a new
method that simultaneously estimates changes in population size
and in the ratio of male to female effective sizes. Finally, they
apply the new method to several human populations.</li>
</ul></li>
<li><p>Jun 29 <a href="https://advances.sciencemag.org/content/6/8/eaay5483">Rogers, Harris, and Achenbach. 2020.</a>
Neanderthal-Denisovan ancestors interbred with a distantly-related
hominin. Presenter: Alan Rogers</p>

<ul>
<li>"We show that early in the middle Pleistocene, long before the
expansion of modern humans into Eurasia, the 'neandersovan'
ancestors of Neanderthals and Denisovans undertook a very similar
expansion. In both cases, an African population expanded into
Eurasia, endured a narrow bottleneck of population size, interbred
with indigeneous Eurasians, largely replaced them, and split into
eastern and western sub-populations. In the earlier expansion,
neandersovans interbred with a 'superarchaic' population that had
been separate since about 2 mya and may represent the original
expansion of humans into Eurasia."</li>
</ul></li>
<li><p>Jul 06 Two readings: (1) <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0186237">Munung et al. 2017</a> Equity in
international health research collaborations in Africa: Perceptions
and expectations of African researchers, and (2)
<a href="https://advances.sciencemag.org/content/5/10/eaaw7238.abstract">Hoppe et al. 2019.</a> Topic choice contributes to the
lower rate of NIH awards to African-American/black
scientists. Presenter: Tina Lasisi and Tim Webster. </p></li>
<li><p>Jul 13 <a href="https://www.sciencedirect.com/science/article/pii/S0959437X1730182X?via%3Dihub">Henn, Steele, and Weaver. 2018.</a> Clarifying
distinct models of modern human origins in Africa. Presenter:
Brenna Henn.</p>

<ul>
<li>"We describe four possible models for the origins of Homo sapiens
in Africa based on published literature from paleoanthropology and
human genetics. We briefly outline expectations for data patterns
under each model, with a special focus on genetic data." </li>
</ul></li>
<li><p>Jul 20 <a href="https://www.biorxiv.org/content/10.1101/2020.03.13.990523v1">Peter. 2020</a> 100,000 years of gene flow between
Neandertals and Denisovans in the Altai mountains. Presenter:
Benjamin Peter.</p>

<ul>
<li>Studies gene flow between Neanderthals and Denisovans using a new
method that can cope with degraded, low-coverage, DNA. Finds
evidence of gene flow in both directions over a long period of
time. But only in the Altai region of Siberia. European
Neanderthals show no evidence of Denisovan admixture.</li>
</ul></li>
<li><p>Jul 27 <a href="https://www.biorxiv.org/content/10.1101/2020.06.01.127555v1">Cole et al. 2020.</a> Ancient admixture into
Africa from the ancestors of non-Africans. Presenter: Chris Cole. </p>

<ul>
<li>"Here we present SMCSMC, a Bayesian method for inference of
time-varying population sizes and directional migration rates
under the coalescent-with-recombination model, to study ancient
demographic events. We find evidence for substantial migration
from the ancestors of present-day Eurasians into African groups
between 40 and 70 thousand years ago, predating the divergence of
Eastern and Western Eurasian lineages. This event accounts for
previously unexplained genetic diversity in African populations,
and supports the existence of novel population substructure in the
Late Middle Paleolithic."</li>
</ul></li>
<li><p>Aug 03 <a href="https://www.nature.com/articles/s41586-020-2487-2">Ioannidis et al. 2020</a> Native American gene flow
into Polynesia predating Easter Island settlement. Presenter: Alan
Rogers.</p>

<ul>
<li>"Here we analyse genome-wide variation in individuals from
islands across Polynesia for signs of Native American admixture,
analysing 807 individuals from 17 island populations and 15
Pacific coast Native American groups. We find conclusive evidence
for prehistoric contact of Polynesian individuals with Native
American individuals (around ad 1200) contemporaneous with the
settlement of remote Oceania. Our analyses suggest
strongly that a single contact event occurred in eastern
Polynesia, before the settlement of Rapa Nui, between Polynesian
individuals and a Native American group most closely related to
the indigenous inhabitants of present-day Colombia." </li>
</ul></li>
<li><p>Aug 10 <a href="https://www.biorxiv.org/content/10.1101/2020.06.25.170308v1.full?%3Fcollection=">Chakraborty et al. 2020.</a> Whole genomes
reveal severe bottleneck among Asian hunter-gatherers following the
invention of agriculture. Presenter: Saikat Chakraborty.</p>

<ul>
<li>"Using genomic data, we show that for the majority of Asian
hunter-gatherers, population-sizes drastically decreased following
the invention of agriculture. We argue that a combination of
displacement, enslavement and disease resulted in the decimation
of hunter-gatherer societies."</li>
</ul></li>
<li><p>Aug 17 <a href="https://academic.oup.com/mbe/article/doi/10.1093/molbev/msaa140/5874945?fbclid=IwAR3FN-AN8t7pFfz3OMNrC_22JA1bzrLPqSQswSoWqMjhZz3T_P3mwANGWTw#.XxmB88-hF6o.facebook">Schlebusch et al. 2020</a> Khoe-San genomes reveal
unique variation and confirm the deepest population divergence in
<em>Homo sapiens</em>. Presenter: Carina Schlebusch.</p>

<ul>
<li>Sequenced 25 full genomes from Koe-San populations in southern
Africa. Find that Koe-San have more genetic diversity than any
other population and that 25% of variants are unique to Koe-San.
Evidence for selection at genes involved in immunity, the brain,
sperm, diet, and muscle. Evidence of a decline in population size
100,000 years ago.</li>
</ul></li>
</ul>

<h2>Fall 2020</h2>

<ul>
<li><p>Sep 14 <a href="https://advances.sciencemag.org/content/6/24/eaaz0183.abstract">Wang et al. 2020.</a> Ancient genomes reveal complex
patterns of population movement, interaction, and replacement in
sub-Saharan Africa. Presenter: Ke Wang and Stephan Schiffels.</p>

<ul>
<li>"We report genome-wide data from 20 ancient sub-Saharan African
individuals, including the first reported ancient DNA from the
DRC, Uganda, and Botswana. These data demonstrate the
contraction of diverse, once contiguous hunter-gatherer
populations, and suggest the resistance to interaction with
incoming pastoralists of delayed-return foragers in aquatic
environments. ... In Botswana, we show that Bantu ancestry
post-dates admixture between pastoralists and foragers,
suggesting an earlier spread of pastoralism than farming to
southern Africa."</li>
</ul></li>
<li><p>Sep 21 <a href="https://www.sciencedirect.com/science/article/pii/S0092867419309547?casa_token=ELt7WwOV6moAAAAA:a_t_5Kb2N51gqrDjr3qKtLkCYyc1OVzIybi78NBVrbY0K9c0xHB8kJQW8sLF5c0wopUCcSo-y">Gokhman, et al. 2019</a> Reconstructing Denisovan
anatomy using DNA methylation maps. Presenter: David Gokhman.</p>

<ul>
<li>"We present a method for reconstructing skeletal morphology using
DNA methylation patterns. We suggest that Denisovans likely shared
with Neanderthals traits such as an elongated face and a wide
pelvis. We also identify Denisovan-derived changes, such as an
increased dental arch and lateral cranial expansion. Our predictions
match the only morphologically informative Denisovan bone to date,
as well as the Xuchang skull, which was suggested by some to be a
Denisovan. We conclude that DNA methylation can be used to
reconstruct anatomical features, including some that do not survive
in the fossil record."</li>
</ul></li>
<li><p>Sep 28 <a href="https://science.sciencemag.org/content/367/6484/eaay5012.abstract?casa_token=8u4TIciulh8AAAAA:t_v5lDpHNE9v9sliSlZhzfwuvvg1bhLyyGkBrV7fJl-tbYgtgI0CAEVrqJWSPhBsjydI6PqXBVQR">Bergstrom et al. 2020</a> Insights into human genetic
variation and population history from 929 diverse
genomes. Presenter: Anders Bergstrom.</p>

<ul>
<li>A study of 929 high coverage genomes from 54 diverse populations.
"Populations in central and southern Africa, the Americas, and
Oceania each harbor tens to hundreds of thousands of private,
common genetic variants. Most of these variants arose as new
mutations rather than through archaic introgression, except in
Oceanian populations, where many private variants derive from
Denisovan admixture. ... We estimate that the genetic separation
between present-day human populations occurred mostly within the
past 250,000 years. However, these early separations were gradual
in nature and shaped by protracted gene flow. ... Most populations
expanded in size over the past 10,000 years, but hunter-gatherer
groups did not. ... The low diversity among the Neanderthal
haplotypes segregating in present-day populations indicates that,
while more than one Neanderthal individual must have contributed
genetic material to modern humans, there was likely only one major
episode of admixture. By contrast, Denisovan haplotype diversity
reflects a more complex history involving more than one episode of
admixture."</li>
</ul></li>
<li><p>Oct 05 <a href="https://academic.oup.com/mbe/article/35/1/107/4554431">O'Toole, Hurst, and McLysaught. 2018</a> Faster
evolving primate genes are more likely to duplicate. Presenter: Alan
Rogers with Laurence Hurst.</p>

<ul>
<li>"An attractive and long-standing hypothesis regarding the evolution
of genes after duplication posits that the duplication event
creates new evolutionary possibilities by releasing a copy of the
gene from constraint. Apparent support was found in numerous
analyses, particularly, the observation of higher rates of
evolution in duplicated as compared with singleton genes. Could it,
instead, be that more duplicable genes (owing to mutation,
fixation, or retention biases) are intrinsically faster evolving?
To uncouple the measurement of rates of evolution from the
determination of duplicate or singleton status, we measure the
rates of evolution in singleton genes in outgroup primate lineages
but classify these genes as to whether they have duplicated or not
in a crown group of great apes. We find that rates of evolution are
higher in duplicable genes prior to the duplication event. In part
this is owing to a negative correlation between coding sequence
length and rate of evolution, coupled with a bias toward smaller
genes being more duplicable. The effect is masked by difference in
expression rate between duplicable genes and
singletons. Additionally, in contradiction to the classical
assumption, we find no convincing evidence for an increase in dN/dS
after duplication, nor for rate asymmetry between duplicates. We
conclude that high rates of evolution of duplicated genes are not
solely a consequence of the duplication event, but are rather a
predictor of duplicability. These results are consistent with a
model in which successful gene duplication events in mammals are
skewed toward events of minimal phenotypic impact."</li>
</ul></li>
<li><p>Oct 12 <a href="https://www.nature.com/articles/s41559-020-1261-z">Rinker et al. 2020</a> Neanderthal introgression
reintroduced functional ancestral alleles lost in Eurasian
populations. Presenter: Tony Capra. If you have trouble getting
access to this paper, try the
<a href="https://www.biorxiv.org/content/biorxiv/early/2019/11/15/533257.full.pdf">version</a>
on bioRxiv.</p>

<ul>
<li>"Our study reveals that Neanderthal introgression reintroduced
thousands of lost ancestral variants with gene regulatory activity
and that these RAs [reintroduced alleles] were more tolerated than
NDAs [Neanderthal-derived alleles]. Thus, RAs and their distinct
evolutionary histories must be considered when evaluating the
effects of introgression."</li>
</ul></li>
<li><p>Oct 19 <a href="https://www.nature.com/articles/s41588-020-0646-x">Porubsky et al. 2020</a> Recurrent inversion toggling
and great ape genome evolution. Presenter: David Porubsky.</p>

<ul>
<li>"the X chromosome is most enriched (2.5-fold) for inversions, on
the basis of its size and duplication content. There is an excess
of differentially expressed primate genes near the breakpoints of
large (>100 kilobases (kb)) inversions but not smaller events. We
show that when great ape lineage-specific duplications emerge,
they preferentially (approximately 75%) occur in an inverted
orientation compared to that at their ancestral locus. We
construct megabase-pair scale haplotypes for individual
chromosomes and identify 23 genomic regions that have recurrently
toggled between a direct and an inverted state over 15 million
years. The direct orientation is most frequently the derived state
for human polymorphisms that predispose to recurrent copy number
variants associated with neurodevelopmental disease." </li>
</ul></li>
<li><p>Oct 26 <a href="https://journals.plos.org/plosgenetics/article?id=10.1371%2Fjournal.pgen.1008895&amp;fbclid=IwAR0sx3T1_92BQHUShlaY_oBV3LK3yZbgOz6K-RI6Pce4fIYUgSbaRDkemE0">Hubisz, Williams, and Siepel. 2020</a> Mapping gene
flow between ancient hominins through demography-aware inference of
the ancestral recombination graph. Presenter: Melissa Hubisz</p>

<ul>
<li>"We present a major extension of the ARGweaver algorithm, called
ARGweaver-D, which can infer local genetic relationships under a
user-defined demographic model that includes population splits and
migration events. ... We identify 3% of the Neanderthal genome
that is putatively introgressed from ancient humans, and estimate
that the gene flow occurred between 200-300kya. We find no
convincing evidence that negative selection acted against these
regions. Finally, we predict that 1% of the Denisovan genome was
introgressed from an unsequenced, but highly diverged, archaic
hominin ancestor. About 15% of these “super-archaic”
regions—comprising at least about 4Mb—were, in turn, introgressed
into modern humans and continue to exist in the genomes of people
alive today." </li>
</ul></li>
<li><p>Nov 02 <a href="https://www.genetics.org/content/early/2020/08/24/genetics.120.303622">Galtier and Rouselle. 2020</a>;
<a href="https://www.biorxiv.org/content/10.1101/861849v4">bioRxiv version</a>;
<a href="https://evolbiol.peercommunityin.org/public/rec?id=228">PCI Evol Biol recommendation</a>. How much does
Ne vary among species?  Presenter: Nicolas Galtier</p>

<ul>
<li>"We assess the magnitude of variation in drift power among species
of animals via its effect on the mutation load. ... We estimate
that the power of drift varies by a factor of at least 500 between
large-Ne and small-Ne species of animals, i.e., an order of
magnitude more than the among-species variation in genetic
diversity."</li>
</ul></li>
<li><p>Nov 09 Cancelled</p></li>
<li><p>Nov 16 <a href="https://www.genetics.org/content/215/2/497.abstract">Tashkent et al. 2020</a> Analysis of haplotypic
variation and deletion polymorphisms point to multiple archaic
introgression events, including from Altai Neanderthal
lineage. Presenters: Omer Gokcumen and Ozgur Taskent.</p>

<ul>
<li>"We present a comprehensive analysis of the unusually divergent
haplotypes in the Eurasian genomes and show that they can be
traced back to multiple introgression events. In parallel, we
document hundreds of deletion polymorphisms shared with
Neanderthals. A locus-specific analysis of one such shared
deletion suggests the existence of a direct introgression event
from the Altai Neanderthal lineage into the ancestors of extant
East Asian populations."</li>
</ul></li>
<li><p>Nov 23 Cancelled</p></li>
<li><p>Nov 30 <a href="https://science.sciencemag.org/content/369/6511/1653.full">Petr et al. 2020</a> <a href="https://www.biorxiv.org/content/10.1101/2020.03.09.983445v1.abstract">bioRxiv</a>
The evolutionary history of Neandertal and Denisovan Y
chromosomes. Presenters: Martin Petr and Janet Kelso.</p>

<ul>
<li>"We find that the Denisovan Y chromosomes split around 700
thousand years ago (kya) from a lineage shared by Neandertal and
modern human Y chromosomes, which diverged from each other around
370 kya. The phylogenetic relationships of archaic and modern
human Y chromosomes therefore differ from population relationships
inferred from their autosomal genomes, and mirror the
relationships observed on the level of mitochondrial DNA. This
provides strong evidence that gene flow from an early lineage
related to modern humans resulted in the replacement of both the
mitochondrial and Y chromosomal gene pools in late
Neandertals. Although unlikely under neutrality, we show that this
replacement is plausible if the low effective population size of
Neandertals resulted in an increased genetic load in their Y
chromosomes and mitochondrial DNA relative to modern humans." </li>
</ul></li>
<li><p>Dec 07 <a href="https://www.nature.com/articles/s41586-020-2688-8">Margaryan et al. 2020</a> Population genomics of the
Viking world. Presenter: Ashot Margaryan.</p>

<ul>
<li>"The maritime expansion of Scandinavian populations during the
Viking Age (about ad 750–1050) was a far-flung transformation in
world history1,2. Here we sequenced the genomes of 442 humans from
archaeological sites across Europe and Greenland (to a median
depth of about 1×) to understand the global influence of this
expansion. We find the Viking period involved gene flow into
Scandinavia from the south and east. We observe genetic structure
within Scandinavia, with diversity hotspots in the south and
restricted gene flow within Scandinavia. We find evidence for a
major influx of Danish ancestry into England; a Swedish influx
into the Baltic; and Norwegian influx into Ireland, Iceland and
Greenland. Additionally, we see substantial ancestry from
elsewhere in Europe entering Scandinavia during the Viking Age."</li>
</ul></li>
</ul>
