<h1>Suggested readings</h1>

<ol>
<li><p><a href="https://www.nature.com/articles/s41586-020-2688-8">Margaryan et al. 2020</a> Population genomics of the
Viking world.</p>

<ul>
<li>"The maritime expansion of Scandinavian populations during the
Viking Age (about ad 750–1050) was a far-flung transformation in
world history1,2. Here we sequenced the genomes of 442 humans from
archaeological sites across Europe and Greenland (to a median
depth of about 1×) to understand the global influence of this
expansion. We find the Viking period involved gene flow into
Scandinavia from the south and east. We observe genetic structure
within Scandinavia, with diversity hotspots in the south and
restricted gene flow within Scandinavia. We find evidence for a
major influx of Danish ancestry into England; a Swedish influx
into the Baltic; and Norwegian influx into Ireland, Iceland and
Greenland. Additionally, we see substantial ancestry from
elsewhere in Europe entering Scandinavia during the Viking Age."</li>
</ul></li>
<li><p><a href="https://www.g3journal.org/content/8/8/2805.short">Brandt et al 2018</a> The Effect of Balancing Selection
on Population Differentiation: A Study with HLA Genes.</p>

<ul>
<li>"Theory predicts that balancing selection reduces population
differentiation, as measured by FST...However, in pairs of closely
related populations, where genome-wide differentiation is low,
differentiation at HLA is higher than in other genomic
regions. Such increased population differentiation at HLA genes
for recently diverged population pairs was reproduced in
simulations of overdominant selection, as long as the fitness of
the homozygotes differs between the diverging populations." </li>
</ul></li>
<li><p><a href="https://academic.oup.com/mbe/article/37/1/183/5570178">van der Valk et al 2019</a> The genome of the
endangered Dryas monkey provides new insights into the evolutionary
history of the Vervets.</p>

<ul>
<li>"We show that the [endangered] Dryas monkey represents a sister
lineage to the vervets (Chlorocebus sp.) and has diverged from
them ∼1.4 Ma with additional bidirectional gene flow
∼750,000–500,000 years ago that has likely involved the crossing
of the Congo River. ... We find high genetic diversity and low
levels of inbreeding and genetic load in the studied Dryas monkey
individual. This suggests that the current population carries
sufficient genetic variability for long-term survival and might be
larger than currently recognized."</li>
</ul></li>
<li><p><a href="https://www.nature.com/articles/s41562-020-0897-7">Betti et al. 2020</a> Climate shaped how Neolithic
farmers and European hunter-gatherers interacted after a major
slowdown from 6,100 BCE to 4,500 BCE.</p>

<ul>
<li>"We identify four axes of [European Neolithic] expansion and
observe a slowdown along three axes when crossing the same
climatic threshold. This threshold reflects the quality of the
growing season, suggesting that Near Eastern crops might have
struggled under more challenging climatic conditions. This same
threshold also predicts the mixing of farmers and hunter-gatherers
as estimated from ancient DNA, suggesting that unreliable yields
in these regions might have favoured the contact between the two
groups."</li>
</ul></li>
<li><p><a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1008552">Wang et al. 2020</a> Tracking human population structure
through time from whole genome sequences.</p>

<ul>
<li>These authors introduce a new method for inferring the deep
history of population size and admixture. Using this method, they
"detect traces of extremely deep ancestry between some African
populations, with around 1% of ancestry dating to divergences older
than a million years ago."</li>
</ul></li>
<li><p><a href="https://onlinelibrary.wiley.com/doi/full/10.1111/mec.13154?casa_token=z0DDFCt5kyEAAAAA%3AQGtGPXOttZnSydrX6XabMB5ODbnggn7PCrkOT1wItwcMDo82iv5tyfXfYStld5oL5FhgKw4EZENa">Peischi and Excoffier. 2015</a> Expansion load:
recessive mutations and the role of standing genetic variation.</p>

<ul>
<li>"Expanding populations incur a mutation burden--the so‐called
expansion load. ... [We find that] the loss of deleterious mutations
at some loci is compensated by an increase of their frequencies at
other loci. The frequency of deleterious homozygotes therefore
increases along the expansion axis, whereas the average number of
deleterious mutations per individual remains nearly constant across
the species range."</li>
</ul></li>
<li><p><a href="https://www.nature.com/articles/s41598-020-59110-3">Yin et al. 2020</a> The draft genome of mandrill
(Mandrillus sphinx): An Old World monkey.</p>

<ul>
<li>"We found that 17 genes in the mandrill that have been associated
with disease phenotypes in human such as Lung cancer, cranial
volume and asthma, barbored amino acids changing mutations. Gene
family analyses revealed expansion of several genes, and several
genes associated with stress environmental adaptation and innate
immunity responses exhibited signatures of positive selection. In
summary, we established the first draft genome of  the mandrill of
value for studies on evolution and human health." </li>
</ul></li>
<li><p><a href="https://www.nature.com/articles/s41467-019-12174-w">He et al. 2019</a> Long-read assembly of the Chinese rhesus
macaque genome and identification of ape-specific structural
variants. </p>

<ul>
<li>"We present a high-quality de novo genome assembly (rheMacS) of
the Chinese rhesus macaque (Macaca mulatta) using long-read
sequencing and multiplatform scaffolding approaches. Compared to
the current Indian rhesus macaque reference genome (rheMac8),
rheMacS increases sequence contiguity 75-fold, closing 21,940 of
the remaining assembly gaps (60.8 Mbp). We improve gene annotation
by generating more than two million full-length transcripts from
ten different tissues by long-read RNA sequencing. We sequence
resolve 53,916 structural variants (96% novel) and identify 17,000
ape-specific structural variants (ASSVs) based on comparison to
ape genomes." </li>
</ul></li>
<li><p><a href="https://www.biorxiv.org/content/10.1101/851188v1">Immel et al. 2019.</a> Neolithic genomes reveal a
distinct ancient HLA allele pool and population transformation in
Europe.</p>

<ul>
<li>Studied 42 late-Neolithic genomes from the Wartenberg culture of
Europe. A high percentage--40%--of their ancestry was from hunter
gatherers, and their HLA alleles were unlike those of modern
Europeans. The authors argue that their immune systems were
specialized for defense against viruses.</li>
</ul></li>
<li><p><a href="https://www.biorxiv.org/content/10.1101/427583v2?%3Fcollection=">Zen, Huber, Davies, and Lohmuller, 2020</a> Greater
strength of selection and higher proportion of beneficial amino
acid changing mutations in humans compared to mice and Drosophila
melanogaster.</p>

<ul>
<li>"We use improved modeling of weakly deleterious mutations and the
demographic history of the outgroup species and ancestral
population and estimate that at least 20% of nonsynonymous
substitutions between humans and an outgroup species were fixed by
positive selection. This estimate is much higher than previous
estimates, which did not correct for the sizes of the outgroup
species and ancestral population."</li>
</ul></li>
<li><p><a href="https://www.nature.com/articles/s41467-020-18418-4">Byrne et al. 2020</a> Dutch population structure across
space, time and GWAS design.</p>

<ul>
<li>The authors use haplotype sharing to estimate the fine-grained
population structure and history of the Dutch population. "Despite
superexponential population growth, regional demographic estimates
reveal population crashes contemporaneous with the Black
Death. Within Dutch and international data, GWAS incorporating
fine-grained haplotypic covariates are less confounded than
standard methods."</li>
</ul></li>
<li><p><a href="https://www.nature.com/articles/s41437-020-0345-5">Parreira et al. 2020</a> Genetic consequences of social
structure in the golden-crowned sifaka.</p>

<ul>
<li>"We observe high excess of heterozygotes within
[social groups (SGs)]. ... While offspring are characterised by a
high excess of heterozygotes (FIS &lt; 0), the reproductive pair does
not show dramatic departures from Hardy–Weinberg
expectations. Moreover, the heterozygosity excess disappears at
larger geographic scales (sites and forests). ... Our results
suggest that social structure leads to high levels of outbreeding
without the need for active inbreeding avoidance mechanisms."</li>
</ul></li>
<li><p><a href="https://www.biorxiv.org/content/10.1101/2020.07.03.186296v1">Zeberg and Paabo. 2020</a> The major genetic risk
factor for severe COVID-19 is inherited from Neandertals.</p>

<ul>
<li>"A recent genetic association study (Ellinghaus et al. 2020)
identified a gene cluster on chromosome 3 as a risk locus for
respiratory failure in SARS-CoV-2. Recent data comprising 3,199
hospitalized COVID-19 patients and controls reproduce this and
find that it is the major genetic risk factor for severe
SARS-CoV-2 infection and hospitalization (COVID-19 Host Genetics
Initiative). Here, we show that the risk is conferred by a genomic
segment of ~50 kb that is inherited from Neandertals and occurs at
a frequency of ~30% in south Asia and ~8% in Europe."</li>
</ul></li>
<li><p><a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msaa119/5841671">Zeberg, et al. 2020.</a> The Neandertal Progesterone
Receptor.</p>

<ul>
<li>"The gene encoding the progesterone receptor (PGR) carries
introgressed Neandertal haplotypes...[, which] have reached nearly
20% frequency in non-Africans and have been associated with
preterm birth...  Present-day carriers of the Neandertal
haplotypes express higher levels of the receptor. In a cohort of
present-day Britons, these carriers have more siblings, fewer
miscarriages, and less bleeding during early pregnancy suggesting
that the Neandertal progesterone receptor alleles promote
fertility. This may explain their high frequency in modern human
populations."</li>
</ul></li>
<li><p><a href="https://academic.oup.com/mbe/article/34/10/2704/3988100">Xu et al. 2017.</a> Archaic hominin introgression in Africa
contributes to functional salivary MUC7 genetic variation.</p>

<ul>
<li>Unlike previous authors, these authors find no association between
the MUC7 salivary protein and asthma. On the other hand, this
locus is associated with the oral microbiome. Furthermore, one
haplotype within this locus seems to have originated in an unknown
but deeply-divergent (superarchaic?) population, which
introgressed into the ancestors of modern Africans.</li>
</ul></li>
<li><p><a href="https://www.biorxiv.org/content/10.1101/2020.04.16.045450v1">Hernandez, Shenk, and Perry. 2020</a> Factors
influencing taxonomic unevenness in scientific research: A
mixed-methods case study of non-human primate genomic sequence data
generation.</p>

<ul>
<li>How do scientists decide which species to focus on in
DNA-sequencing studies? This study shows that the most-studied
species are those for which there is a large prior literature and
that there is a bias in favor of species closely-related to our
own.</li>
</ul></li>
<li><p><a href="https://www.nature.com/articles/s41467-018-04204-w">Hicks et al. 2018.</a> Gut microbiomes of wild great apes
fluctuate seasonally in response to diet.</p>

<ul>
<li>"Here we use 16S rRNA sequencing to characterize the microbiota of
wild western lowland gorillas and sympatric central chimpanzees
and demonstrate compositional divergence between the microbiotas
of gorillas, chimpanzees, Old World monkeys, and modern humans. We
show that gorilla and chimpanzee microbiomes fluctuate with
seasonal rainfall patterns and frugivory."</li>
</ul></li>
<li><p><a href="https://www.nature.com/articles/s41598-020-66867-0">Cawthon et al. 2020.</a> Germline mutation rates in
young adults predict longevity and reproductive lifespan.</p>

<ul>
<li>"The paper is the first report showing that differences in the
levels of mutation accumulation in young healthy adults predict
remaining lifespan in both sexes, and remaining reproductive span
in women. It is also the first report showing that mutation
accumulation accelerates in females after menarche, and it
presents data suggesting that mutation accumulation accelerates in
males after puberty as well. These results support two
long-standing theories about the nature of aging: 1) aging is due,
at least in part, to mutation accumulation in adults contributing
to cell and tissue dysfunction, onset of chronic diseases, and
ultimately death; and 2) aging begins in adolescence, possibly
triggered by puberty, consistent with the evolutionary biology
theory that the force of natural selection to maintain robust
health should begin to decline once the reproductive stage of life
is attained."</li>
</ul></li>
<li><p><a href="https://www.nature.com/articles/s41559-020-1102-0">Fernandes et al. 2020</a> The spread of steppe and
Iranian-related ancestry in the islands of the western
Mediterranean.</p>

<ul>
<li>"We generated genome-wide ancient-DNA data from the Balearic
Islands, Sicily and Sardinia, increasing the number of individuals
with reported data from 5 to 66. The oldest individual from the
Balearic Islands (~2400 bc) carried ancestry from steppe
pastoralists that probably derived from west-to-east migration from
Iberia, although two later Balearic individuals had less ancestry
from steppe pastoralists. In Sicily, steppe pastoralist ancestry
arrived by ~2200 bc. ... Major immigration into Sardinia began in
the first millennium bc and, at present, no more than 56–62% of
Sardinian ancestry is from its first farmers. This value is lower
than previous estimates, highlighting that Sardinia, similar to
every other region in Europe, has been a stage for major movement
and mixtures of people." </li>
</ul></li>
<li><p><a href="https://science.sciencemag.org/content/368/6498/1495.abstract">Sinding et al. 2020.</a> Arctic-adapted dogs emerged at
the Pleistocene-Holocene transition.</p>

<ul>
<li>"This analysis indicates that sled dogs represent an ancient
lineage going back at least 9500 years and that wolves bred with
the ancestors of sled dogs and precontact American dogs. However,
gene flow between sled dogs and wolves likely stopped before ∼9500
years ago."</li>
</ul></li>
</ol>
