<h1>Suggested readings</h1>

<ol>
<li><p><a href="https://www.sciencedirect.com/science/article/pii/S0092867419309547?casa_token=ELt7WwOV6moAAAAA:a_t_5Kb2N51gqrDjr3qKtLkCYyc1OVzIybi78NBVrbY0K9c0xHB8kJQW8sLF5c0wopUCcSo-yg">Gokhman, et al. 2019</a> Reconstructing Denisovan
Anatomy Using DNA Methylation Maps.</p>

<ul>
<li>"We present a method for reconstructing skeletal morphology using
DNA methylation patterns. We suggest that Denisovans likely shared
with Neanderthals traits such as an elongated face and a wide
pelvis. We also identify Denisovan-derived changes, such as an
increased dental arch and lateral cranial expansion. Our predictions
match the only morphologically informative Denisovan bone to date,
as well as the Xuchang skull, which was suggested by some to be a
Denisovan. We conclude that DNA methylation can be used to
reconstruct anatomical features, including some that do not survive
in the fossil record."</li>
</ul></li>
<li><p><a href="https://academic.oup.com/mbe/article/35/1/107/4554431">O'Toole, Hurst, and McLysaught. 2018</a> Faster
evolving primate genes are more likely to duplicate.</p>

<ul>
<li>"An attractive and long-standing hypothesis regarding the evolution
of genes after duplication posits that the duplication event
creates new evolutionary possibilities by releasing a copy of the
gene from constraint. Apparent support was found in numerous
analyses, particularly, the observation of higher rates of
evolution in duplicated as compared with singleton genes. Could it,
instead, be that more duplicable genes (owing to mutation,
fixation, or retention biases) are intrinsically faster evolving?
To uncouple the measurement of rates of evolution from the
determination of duplicate or singleton status, we measure the
rates of evolution in singleton genes in outgroup primate lineages
but classify these genes as to whether they have duplicated or not
in a crown group of great apes. We find that rates of evolution are
higher in duplicable genes prior to the duplication event. In part
this is owing to a negative correlation between coding sequence
length and rate of evolution, coupled with a bias toward smaller
genes being more duplicable. The effect is masked by difference in
expression rate between duplicable genes and
singletons. Additionally, in contradiction to the classical
assumption, we find no convincing evidence for an increase in dN/dS
after duplication, nor for rate asymmetry between duplicates. We
conclude that high rates of evolution of duplicated genes are not
solely a consequence of the duplication event, but are rather a
predictor of duplicability. These results are consistent with a
model in which successful gene duplication events in mammals are
skewed toward events of minimal phenotypic impact."</li>
</ul></li>
<li><p><a href="https://www.genetics.org/content/215/2/497.abstract">Tashkent et al. 2020</a> Analysis of haplotypic
variation and deletion polymorphisms point to multiple archaic
introgression events, including from Altai Neanderthal lineage.</p>

<ul>
<li>"We present a comprehensive analysis of the unusually divergent
haplotypes in the Eurasian genomes and show that they can be
traced back to multiple introgression events. In parallel, we
document hundreds of deletion polymorphisms shared with
Neanderthals. A locus-specific analysis of one such shared
deletion suggests the existence of a direct introgression event
from the Altai Neanderthal lineage into the ancestors of extant
East Asian populations."</li>
</ul></li>
<li><p><a href="https://academic.oup.com/mbe/article/37/1/183/5570178">van der Valk et al 2019</a> The genome of the
endangered Dryas monkey provides new insights into the evolutionary
history of the Vervets.</p>

<ul>
<li>"We show that the [endangered] Dryas monkey represents a sister
lineage to the vervets (Chlorocebus sp.) and has diverged from
them ∼1.4 Ma with additional bidirectional gene flow
∼750,000–500,000 years ago that has likely involved the crossing
of the Congo River. ... We find high genetic diversity and low
levels of inbreeding and genetic load in the studied Dryas monkey
individual. This suggests that the current population carries
sufficient genetic variability for long-term survival and might be
larger than currently recognized."</li>
</ul></li>
<li><p><a href="https://science.sciencemag.org/content/367/6484/eaay5012.abstract?casa_token=8u4TIciulh8AAAAA:t_v5lDpHNE9v9sliSlZhzfwuvvg1bhLyyGkBrV7fJl-tbYgtgI0CAEVrqJWSPhBsjydI6PqXBVQR">Bergstrom et al. 2020</a> Insights into human genetic
variation and population history from 929 diverse genomes.</p>

<ul>
<li>A study of 929 high coverage genomes from 54 diverse populations.
"Populations in central and southern Africa, the Americas, and
Oceania each harbor tens to hundreds of thousands of private,
common genetic variants. Most of these variants arose as new
mutations rather than through archaic introgression, except in
Oceanian populations, where many private variants derive from
Denisovan admixture. ... We estimate that the genetic separation
between present-day human populations occurred mostly within the
past 250,000 years. However, these early separations were gradual
in nature and shaped by protracted gene flow. ... Most populations
expanded in size over the past 10,000 years, but hunter-gatherer
groups did not. ... The low diversity among the Neanderthal
haplotypes segregating in present-day populations indicates that,
while more than one Neanderthal individual must have contributed
genetic material to modern humans, there was likely only one major
episode of admixture. By contrast, Denisovan haplotype diversity
reflects a more complex history involving more than one episode of
admixture."</li>
</ul></li>
<li><p><a href="https://journals.plos.org/plosgenetics/article?id=10.1371%2Fjournal.pgen.1008895&amp;fbclid=IwAR0sx3T1_92BQHUShlaY_oBV3LK3yZbgOz6K-RI6Pce4fIYUgSbaRDkemE0">Hubisz, Williams, and Siepel. 2020</a> Mapping gene
flow between ancient hominins through demography-aware inference of
the ancestral recombination graph. </p>

<ul>
<li>"We present a major extension of the ARGweaver algorithm, called
ARGweaver-D, which can infer local genetic relationships under a
user-defined demographic model that includes population splits and
migration events. ... We identify 3% of the Neanderthal genome
that is putatively introgressed from ancient humans, and estimate
that the gene flow occurred between 200-300kya. We find no
convincing evidence that negative selection acted against these
regions. Finally, we predict that 1% of the Denisovan genome was
introgressed from an unsequenced, but highly diverged, archaic
hominin ancestor. About 15% of these “super-archaic”
regions—comprising at least about 4Mb—were, in turn, introgressed
into modern humans and continue to exist in the genomes of people
alive today." </li>
</ul></li>
<li><p><a href="https://www.nature.com/articles/s41562-020-0897-7">Betti et al. 2020</a> Climate shaped how Neolithic
farmers and European hunter-gatherers interacted after a major
slowdown from 6,100 BCE to 4,500 BCE.</p>

<ul>
<li>"We identify four axes of [European Neolithic] expansion and
observe a slowdown along three axes when crossing the same
climatic threshold. This threshold reflects the quality of the
growing season, suggesting that Near Eastern crops might have
struggled under more challenging climatic conditions. This same
threshold also predicts the mixing of farmers and hunter-gatherers
as estimated from ancient DNA, suggesting that unreliable yields
in these regions might have favoured the contact between the two
groups."</li>
</ul></li>
<li><p><a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1008552">Wang et al. 2020</a> Tracking human population structure
through time from whole genome sequences.</p>

<ul>
<li>These authors introduce a new method for inferring the deep
history of population size and admixture. Using this method, they
"detect traces of extremely deep ancestry between some African
populations, with around 1% of ancestry dating to divergences older
than a million years ago."</li>
</ul></li>
<li><p><a href="https://www.nature.com/articles/s41588-020-0646-x">Porubsky et al. 2020</a> Recurrent inversion toggling
and great ape genome evolution.</p>

<ul>
<li>"the X chromosome is most enriched (2.5-fold) for inversions, on
the basis of its size and duplication content. There is an excess
of differentially expressed primate genes near the breakpoints of
large (>100 kilobases (kb)) inversions but not smaller events. We
show that when great ape lineage-specific duplications emerge,
they preferentially (approximately 75%) occur in an inverted
orientation compared to that at their ancestral locus. We
construct megabase-pair scale haplotypes for individual
chromosomes and identify 23 genomic regions that have recurrently
toggled between a direct and an inverted state over 15 million
years. The direct orientation is most frequently the derived state
for human polymorphisms that predispose to recurrent copy number
variants associated with neurodevelopmental disease." </li>
</ul></li>
<li><p><a href="https://www.nature.com/articles/s41559-020-1261-z">Rinker et al. 2020</a> Neanderthal introgression
reintroduced functional ancestral alleles lost in Eurasian
populations.</p>

<ul>
<li>"Our study reveals that Neanderthal introgression reintroduced
thousands of lost ancestral variants with gene regulatory activity
and that these RAs [reintroduced alleles] were more tolerated than
NDAs [Neanderthal-derived alleles]. Thus, RAs and their distinct
evolutionary histories must be considered when evaluating the
effects of introgression."</li>
</ul></li>
<li><p><a href="https://onlinelibrary.wiley.com/doi/full/10.1111/mec.13154?casa_token=z0DDFCt5kyEAAAAA%3AQGtGPXOttZnSydrX6XabMB5ODbnggn7PCrkOT1wItwcMDo82iv5tyfXfYStld5oL5FhgKw4EZENa">Peischi and Excoffier. 2015</a> Expansion load:
recessive mutations and the role of standing genetic variation.</p>

<ul>
<li>"Expanding populations incur a mutation burden--the so‐called
expansion load. ... [We find that] the loss of deleterious mutations
at some loci is compensated by an increase of their frequencies at
other loci. The frequency of deleterious homozygotes therefore
increases along the expansion axis, whereas the average number of
deleterious mutations per individual remains nearly constant across
the species range."</li>
</ul></li>
<li><p><a href="https://www.genetics.org/content/early/2020/08/24/genetics.120.303622">Galtier and Rouselle. 2020</a> How much does Ne vary
among species?</p>

<ul>
<li>"We assess the magnitude of variation in drift power among species
of animals via its effect on the mutation load. ... We estimate
that the power of drift varies by a factor of at least 500 between
large-Ne and small-Ne species of animals, i.e., an order of
magnitude more than the among-species variation in genetic
diversity."</li>
</ul></li>
<li><p><a href="https://www.nature.com/articles/s41598-020-59110-3">Yin et al. 2020</a> The draft genome of mandrill
(Mandrillus sphinx): An Old World monkey.</p>

<ul>
<li>"We found that 17 genes in the mandrill that have been associated
with disease phenotypes in human such as Lung cancer, cranial
volume and asthma, barbored amino acids changing mutations. Gene
family analyses revealed expansion of several genes, and several
genes associated with stress environmental adaptation and innate
immunity responses exhibited signatures of positive selection. In
summary, we established the first draft genome of  the mandrill of
value for studies on evolution and human health." </li>
</ul></li>
<li><p><a href="https://www.nature.com/articles/s41467-019-12174-w">He et al. 2019</a> Long-read assembly of the Chinese rhesus
macaque genome and identification of ape-specific structural
variants. </p>

<ul>
<li>"We present a high-quality de novo genome assembly (rheMacS) of
the Chinese rhesus macaque (Macaca mulatta) using long-read
sequencing and multiplatform scaffolding approaches. Compared to
the current Indian rhesus macaque reference genome (rheMac8),
rheMacS increases sequence contiguity 75-fold, closing 21,940 of
the remaining assembly gaps (60.8 Mbp). We improve gene annotation
by generating more than two million full-length transcripts from
ten different tissues by long-read RNA sequencing. We sequence
resolve 53,916 structural variants (96% novel) and identify 17,000
ape-specific structural variants (ASSVs) based on comparison to
ape genomes." </li>
</ul></li>
<li><p><a href="https://www.biorxiv.org/content/10.1101/851188v1">Immel et al. 2019.</a> Neolithic genomes reveal a
distinct ancient HLA allele pool and population transformation in
Europe.</p>

<ul>
<li>Studied 42 late-Neolithic genomes from the Wartenberg culture of
Europe. A high percentage--40%--of their ancestry was from hunter
gatherers, and their HLA alleles were unlike those of modern
Europeans. The authors argue that their immune systems were
specialized for defense against viruses.</li>
</ul></li>
<li><p><a href="https://www.biorxiv.org/content/10.1101/427583v2?%3Fcollection=">Zen, Huber, Davies, and Lohmuller, 2020</a> Greater
strength of selection and higher proportion of beneficial amino
acid changing mutations in humans compared to mice and Drosophila
melanogaster.</p>

<ul>
<li>"We use improved modeling of weakly deleterious mutations and the
demographic history of the outgroup species and ancestral
population and estimate that at least 20% of nonsynonymous
substitutions between humans and an outgroup species were fixed by
positive selection. This estimate is much higher than previous
estimates, which did not correct for the sizes of the outgroup
species and ancestral population."</li>
</ul></li>
<li><p><a href="https://www.nature.com/articles/s41467-020-18418-4">Byrne et al. 2020</a> Dutch population structure across
space, time and GWAS design.</p>

<ul>
<li>The authors use haplotype sharing to estimate the fine-grained
population structure and history of the Dutch population. "Despite
superexponential population growth, regional demographic estimates
reveal population crashes contemporaneous with the Black
Death. Within Dutch and international data, GWAS incorporating
fine-grained haplotypic covariates are less confounded than
standard methods."</li>
</ul></li>
<li><p><a href="https://www.nature.com/articles/s41437-020-0345-5">Parreira et al. 2020</a> Genetic consequences of social
structure in the golden-crowned sifaka.</p>

<ul>
<li>"We observe high excess of heterozygotes within
[social groups (SGs)]. ... While offspring are characterised by a
high excess of heterozygotes (FIS &lt; 0), the reproductive pair does
not show dramatic departures from Hardy–Weinberg
expectations. Moreover, the heterozygosity excess disappears at
larger geographic scales (sites and forests). ... Our results
suggest that social structure leads to high levels of outbreeding
without the need for active inbreeding avoidance mechanisms."</li>
</ul></li>
<li><p><a href="https://www.biorxiv.org/content/10.1101/2020.07.03.186296v1">Zeberg and Paabo. 2020</a> The major genetic risk
factor for severe COVID-19 is inherited from Neandertals.</p>

<ul>
<li>"A recent genetic association study (Ellinghaus et al. 2020)
identified a gene cluster on chromosome 3 as a risk locus for
respiratory failure in SARS-CoV-2. Recent data comprising 3,199
hospitalized COVID-19 patients and controls reproduce this and
find that it is the major genetic risk factor for severe
SARS-CoV-2 infection and hospitalization (COVID-19 Host Genetics
Initiative). Here, we show that the risk is conferred by a genomic
segment of ~50 kb that is inherited from Neandertals and occurs at
a frequency of ~30% in south Asia and ~8% in Europe."</li>
</ul></li>
<li><p><a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msaa119/5841671">Zeberg, et al. 2020.</a> The Neandertal Progesterone
Receptor.</p>

<ul>
<li>"The gene encoding the progesterone receptor (PGR) carries
introgressed Neandertal haplotypes...[, which] have reached nearly
20% frequency in non-Africans and have been associated with
preterm birth...  Present-day carriers of the Neandertal
haplotypes express higher levels of the receptor. In a cohort of
present-day Britons, these carriers have more siblings, fewer
miscarriages, and less bleeding during early pregnancy suggesting
that the Neandertal progesterone receptor alleles promote
fertility. This may explain their high frequency in modern human
populations."</li>
</ul></li>
<li><p><a href="https://academic.oup.com/mbe/article/34/10/2704/3988100">Xu et al. 2017.</a> Archaic hominin introgression in Africa
contributes to functional salivary MUC7 genetic variation.</p>

<ul>
<li>Unlike previous authors, these authors find no association between
the MUC7 salivary protein and asthma. On the other hand, this
locus is associated with the oral microbiome. Furthermore, one
haplotype within this locus seems to have originated in an unknown
but deeply-divergent (superarchaic?) population, which
introgressed into the ancestors of modern Africans.</li>
</ul></li>
<li><p><a href="https://www.biorxiv.org/content/10.1101/2020.04.16.045450v1">Hernandez, Shenk, and Perry. 2020</a> Factors
influencing taxonomic unevenness in scientific research: A
mixed-methods case study of non-human primate genomic sequence data
generation.</p>

<ul>
<li>How do scientists decide which species to focus on in
DNA-sequencing studies? This study shows that the most-studied
species are those for which there is a large prior literature and
that there is a bias in favor of species closely-related to our
own.</li>
</ul></li>
<li><p><a href="https://www.nature.com/articles/s41467-018-04204-w">Hicks et al. 2018.</a> Gut microbiomes of wild great apes
fluctuate seasonally in response to diet.</p>

<ul>
<li>"Here we use 16S rRNA sequencing to characterize the microbiota of
wild western lowland gorillas and sympatric central chimpanzees
and demonstrate compositional divergence between the microbiotas
of gorillas, chimpanzees, Old World monkeys, and modern humans. We
show that gorilla and chimpanzee microbiomes fluctuate with
seasonal rainfall patterns and frugivory."</li>
</ul></li>
<li><p><a href="https://www.nature.com/articles/s41598-020-66867-0">Cawthon et al. 2020.</a> Germline mutation rates in
young adults predict longevity and reproductive lifespan.</p>

<ul>
<li>"The paper is the first report showing that differences in the
levels of mutation accumulation in young healthy adults predict
remaining lifespan in both sexes, and remaining reproductive span
in women. It is also the first report showing that mutation
accumulation accelerates in females after menarche, and it
presents data suggesting that mutation accumulation accelerates in
males after puberty as well. These results support two
long-standing theories about the nature of aging: 1) aging is due,
at least in part, to mutation accumulation in adults contributing
to cell and tissue dysfunction, onset of chronic diseases, and
ultimately death; and 2) aging begins in adolescence, possibly
triggered by puberty, consistent with the evolutionary biology
theory that the force of natural selection to maintain robust
health should begin to decline once the reproductive stage of life
is attained."</li>
</ul></li>
<li><p><a href="https://science.sciencemag.org/content/368/6498/1495.abstract">Sinding et al. 2020.</a> Arctic-adapted dogs emerged at
the Pleistocene-Holocene transition.</p>

<ul>
<li>"This analysis indicates that sled dogs represent an ancient
lineage going back at least 9500 years and that wolves bred with
the ancestors of sled dogs and precontact American dogs. However,
gene flow between sled dogs and wolves likely stopped before ∼9500
years ago."</li>
</ul></li>
</ol>
