# Evolutionary Genetics of Humans and Other Primates: A Journal Club

## Introduction

Welcome to eghop, a journal club on the evolutionary genetics of
humans and other primates. We meet weekly by Zoom. During the Fall
semester of 2020, we'll meet Mondays at noon
[Mountain time](https://en.wikipedia.org/wiki/Mountain_Time_Zone).

Each week, we discuss a recent publication or preprint on the
evolutionary genetics of humans or other primates. Everyone is
welcome, from anywhere in the world. You need not be affiliated with
our University, and there is no fee. Undergraduates are welcome.  The
only prerequisite is an interest in the subject. The journal club is
administered by Alan Rogers and Tim Webster, of the University of
Utah.

Participants are expected to treat each other with respect. Anyone who
fails to do so will be blocked.

If you'd like to participate, send email to Alan Rogers
([rogers@anthro.utah.edu](mailto:rogers@anthro.utah.edu)). Unless I
know you already, tell me about yourself. I'll send you the Zoom
meeting id and password.

## Suggested readings

Click [here](suggested.html) for the list of readings that have been
suggested for future meetings of the journal club. Use
[this link][gform] to suggest an article or preprint yourself. We
prefer articles that present new findings--either empirical or
theoretical. Review papers and methodological papers are less likely
to provoke interesting discussions. We'll conduct surveys from time to
time to vote on the suggested readings.

## History

Click [here](past.html) for a history of past journal club meetings.

## Labs recruiting graduate students

Click [here](gradop.html) for a list of labs recruiting graduate
students with an interest in the evolutionary genetics of humans and
other primates.

## Schedule

* Sep 14 [Wang et al. 2020.][wang20:sa] Ancient genomes reveal complex
   patterns of population movement, interaction, and replacement in
   sub-Saharan Africa. Presenter: Ke Wang and Stephan Schiffels.

    * "We report genome-wide data from 20 ancient sub-Saharan African
      individuals, including the first reported ancient DNA from the
      DRC, Uganda, and Botswana. These data demonstrate the
      contraction of diverse, once contiguous hunter-gatherer
      populations, and suggest the resistance to interaction with
      incoming pastoralists of delayed-return foragers in aquatic
      environments. ... In Botswana, we show that Bantu ancestry
      post-dates admixture between pastoralists and foragers,
      suggesting an earlier spread of pastoralism than farming to
      southern Africa."

* Sep 21 [Gokhman, et al. 2019][gokhman20:cell] Reconstructing Denisovan
anatomy using DNA methylation maps. Presenter: David Gokhman.

  * "We present a method for reconstructing skeletal morphology using
  DNA methylation patterns. We suggest that Denisovans likely shared
  with Neanderthals traits such as an elongated face and a wide
  pelvis. We also identify Denisovan-derived changes, such as an
  increased dental arch and lateral cranial expansion. Our predictions
  match the only morphologically informative Denisovan bone to date,
  as well as the Xuchang skull, which was suggested by some to be a
  Denisovan. We conclude that DNA methylation can be used to
  reconstruct anatomical features, including some that do not survive
  in the fossil record."

* Sep 28 [Bergstrom et al. 2020][bergstrom20:s] Insights into human genetic
   variation and population history from 929 diverse
   genomes. Presenter: Anders Bergstrom.

   * A study of 929 high coverage genomes from 54 diverse populations.
   "Populations in central and southern Africa, the Americas, and
   Oceania each harbor tens to hundreds of thousands of private,
   common genetic variants. Most of these variants arose as new
   mutations rather than through archaic introgression, except in
   Oceanian populations, where many private variants derive from
   Denisovan admixture. ... We estimate that the genetic separation
   between present-day human populations occurred mostly within the
   past 250,000 years. However, these early separations were gradual
   in nature and shaped by protracted gene flow. ... Most populations
   expanded in size over the past 10,000 years, but hunter-gatherer
   groups did not. ... The low diversity among the Neanderthal
   haplotypes segregating in present-day populations indicates that,
   while more than one Neanderthal individual must have contributed
   genetic material to modern humans, there was likely only one major
   episode of admixture. By contrast, Denisovan haplotype diversity
   reflects a more complex history involving more than one episode of
   admixture."

* Oct 05 [O'Toole, Hurst, and McLysaught. 2018][otoole18:mbe] Faster
  evolving primate genes are more likely to duplicate. Presenter: Alan
  Rogers with Laurence Hurst.

 * "An attractive and long-standing hypothesis regarding the evolution
   of genes after duplication posits that the duplication event
   creates new evolutionary possibilities by releasing a copy of the
   gene from constraint. Apparent support was found in numerous
   analyses, particularly, the observation of higher rates of
   evolution in duplicated as compared with singleton genes. Could it,
   instead, be that more duplicable genes (owing to mutation,
   fixation, or retention biases) are intrinsically faster evolving?
   To uncouple the measurement of rates of evolution from the
   determination of duplicate or singleton status, we measure the
   rates of evolution in singleton genes in outgroup primate lineages
   but classify these genes as to whether they have duplicated or not
   in a crown group of great apes. We find that rates of evolution are
   higher in duplicable genes prior to the duplication event. In part
   this is owing to a negative correlation between coding sequence
   length and rate of evolution, coupled with a bias toward smaller
   genes being more duplicable. The effect is masked by difference in
   expression rate between duplicable genes and
   singletons. Additionally, in contradiction to the classical
   assumption, we find no convincing evidence for an increase in dN/dS
   after duplication, nor for rate asymmetry between duplicates. We
   conclude that high rates of evolution of duplicated genes are not
   solely a consequence of the duplication event, but are rather a
   predictor of duplicability. These results are consistent with a
   model in which successful gene duplication events in mammals are
   skewed toward events of minimal phenotypic impact."

* Oct 12 [Rinker et al. 2020][rinker20:nee] Neanderthal introgression
   reintroduced functional ancestral alleles lost in Eurasian
   populations. Presenter: Tony Capra.

    * "Our study reveals that Neanderthal introgression reintroduced
      thousands of lost ancestral variants with gene regulatory activity
      and that these RAs [reintroduced alleles] were more tolerated than
      NDAs [Neanderthal-derived alleles]. Thus, RAs and their distinct
      evolutionary histories must be considered when evaluating the
      effects of introgression."

* Oct 19

* Oct 26 [Hubisz, Williams, and Siepel. 2020][hubisz20:plog] Mapping gene
   flow between ancient hominins through demography-aware inference of
   the ancestral recombination graph. Presenter: Melissa Hubisz

  * "We present a major extension of the ARGweaver algorithm, called
    ARGweaver-D, which can infer local genetic relationships under a
    user-defined demographic model that includes population splits and
    migration events. ... We identify 3% of the Neanderthal genome
    that is putatively introgressed from ancient humans, and estimate
    that the gene flow occurred between 200-300kya. We find no
    convincing evidence that negative selection acted against these
    regions. Finally, we predict that 1% of the Denisovan genome was
    introgressed from an unsequenced, but highly diverged, archaic
    hominin ancestor. About 15% of these “super-archaic”
    regions—comprising at least about 4Mb—were, in turn, introgressed
    into modern humans and continue to exist in the genomes of people
    alive today." 

* Nov 02

* Nov 09

* Nov 16

* Nov 23

* Nov 30

* Dec 07

[gform]:
https://docs.google.com/forms/d/e/1FAIpQLSeC-CnDFiBVV5Zw_SR-CrBe5zYSijwkM1aiIqKeDjeQX2eRQQ/viewform?usp=sf_link

[wang20:sa]:
https://advances.sciencemag.org/content/6/24/eaaz0183.abstract

[gokhman20:cell]:
https://www.sciencedirect.com/science/article/pii/S0092867419309547?casa_token=ELt7WwOV6moAAAAA:a_t_5Kb2N51gqrDjr3qKtLkCYyc1OVzIybi78NBVrbY0K9c0xHB8kJQW8sLF5c0wopUCcSo-y

[bergstrom20:s]:
https://science.sciencemag.org/content/367/6484/eaay5012.abstract?casa_token=8u4TIciulh8AAAAA:t_v5lDpHNE9v9sliSlZhzfwuvvg1bhLyyGkBrV7fJl-tbYgtgI0CAEVrqJWSPhBsjydI6PqXBVQR

[otoole18:mbe]:
https://academic.oup.com/mbe/article/35/1/107/4554431

[hubisz20:plog]:
https://journals.plos.org/plosgenetics/article?id=10.1371%2Fjournal.pgen.1008895&fbclid=IwAR0sx3T1_92BQHUShlaY_oBV3LK3yZbgOz6K-RI6Pce4fIYUgSbaRDkemE0

[rinker20:nee]:
https://www.nature.com/articles/s41559-020-1261-z
