# Suggested readings

1. [Immel et al. 2021][immel21:commbiol] Genome-wide study of a
   Neolithic Wartberg grave community reveals distinct HLA variation
   and hunter-gatherer ancestry.

  * "The Wartberg culture (WBC, 3500-2800 BCE) dates to the Late
    Neolithic period, a time of important demographic and cultural
    transformations in western Europe. We performed genome-wide
    analyses of 42 individuals who were interred in a WBC collective
    burial in Niedertiefenbach, Germany (3300-3200cal. BCE). The
    results showed that the farming population of Niedertiefenbach
    carried a surprisingly large hunter-gatherer ancestry com- ponent
    (34–58%). This component was most likely introduced during the
    cultural transformation that led to the WBC. In addition, the
    Niedertiefenbach individuals exhibited a distinct human leukocyte
    antigen gene pool, possibly reflecting an immune response that was
    geared towards detecting viral infections."

1. [Dominguez-Andres et al. 2021][dominguezandres21:biorxiv] Evolution
   of cytokine production capacity in ancient and modern European
   populations. 

  * "We observed that the advent of the Neolithic was a turning point
    for immune-mediated traits in Europeans, favoring those alleles
    linked with the development of tolerance against intracellular
    pathogens and promoting inflammatory responses against
    extracellular microbes." 

1. [Andirko et al. 2021][andirko21:biorxiv] Fine-grained temporal
   mapping of derived high-frequency variants supports the mosaic
   nature of the evolution of Homo sapiens. 

  * These authors summarize published data on the time of origin of
    high-frequency alleles in several categories: those under positive
    selection, those introgressed from archaics, and regulatory genes
    active in the brain. Several of these age distributions are
    bimodal, with one mode at 300-500 kya.

1. [Saag et al 2021][saag21:sa] Genetic ancestry changes in Stone to
   Bronze Age transition in the East European plain.

  * "The transition from Stone to Bronze Age in Central and Western
    Europe was a period of major population movements originating from
    the Ponto-Caspian Steppe. Here, we report new genome-wide sequence
    data from 30 individuals north of this area, from the understudied
    western part of present-day Russia, including 3 Stone Age
    hunter-gatherers (10,800 to 4250 cal BCE) and 26 Bronze Age
    farmers from the Corded Ware complex Fatyanovo Culture (2900 to
    2050 cal BCE). We show that Eastern hunter-gatherer ancestry was
    present in northwestern Russia already from around 10,000
    BCE. Furthermore, we see a change in ancestry with the arrival of
    farming—Fatyanovo Culture individuals were genetically similar to
    other Corded Ware cultures, carrying a mixture of Steppe and
    European early farmer ancestry. Thus, they likely originate from a
    fast migration toward the northeast from somewhere near modern-day
    Ukraine—the closest area where these ancestries coexisted from
    around 3000 BCE."

1. [Vizzari et al 2020][vizzari20:genes] A revised model of
    anatomically modern human expansions out of Africa through a
    machine learning approximate Bayesian computation approach.

  * The authors argue that modern humans emigrated out of Africa in
    two waves, first at 74 kya and later at 46 kya.

1. [Brandt et al 2018][brandt18:g3] The effect of balancing selection
   on population differentiation: a study with HLA genes.

  * "Theory predicts that balancing selection reduces population
    differentiation, as measured by FST...However, in pairs of closely
    related populations, where genome-wide differentiation is low,
    differentiation at HLA is higher than in other genomic
    regions. Such increased population differentiation at HLA genes
    for recently diverged population pairs was reproduced in
    simulations of overdominant selection, as long as the fitness of
    the homozygotes differs between the diverging populations."

1. [Orkin et al 2020][orkin20:biorxiv] The evolution of ecological
   flexibility, large brains, and long lives: capuchin monkey genomics
   revealed with fecalFACS.

  * "We compared genomes of capuchin populations from tropical dry
    forests and lowland rainforests and identified population
    divergence in genes involved in water balance, kidney function,
    and metabolism. Through a comparative genomics approach spanning a
    wide diversity of mammals, we identified genes under positive
    selection associated with longevity and brain development."

1. [van der Valk et al 2019][vanderValk19:mbe] The genome of the
   endangered Dryas monkey provides new insights into the evolutionary
   history of the Vervets.

  * "We show that the [endangered] Dryas monkey represents a sister
    lineage to the vervets (Chlorocebus sp.) and has diverged from
    them ∼1.4 Ma with additional bidirectional gene flow
    ∼750,000–500,000 years ago that has likely involved the crossing
    of the Congo River. ... We find high genetic diversity and low
    levels of inbreeding and genetic load in the studied Dryas monkey
    individual. This suggests that the current population carries
    sufficient genetic variability for long-term survival and might be
    larger than currently recognized."

1. [Betti et al. 2020][betti20:n] Climate shaped how Neolithic
   farmers and European hunter-gatherers interacted after a major
   slowdown from 6,100 BCE to 4,500 BCE.

    * "We identify four axes of [European Neolithic] expansion and
       observe a slowdown along three axes when crossing the same
       climatic threshold. This threshold reflects the quality of the
       growing season, suggesting that Near Eastern crops might have
       struggled under more challenging climatic conditions. This same
       threshold also predicts the mixing of farmers and hunter-gatherers
       as estimated from ancient DNA, suggesting that unreliable yields
       in these regions might have favoured the contact between the two
       groups."

1. [He et al. 2019][he19:nc] Long-read assembly of the Chinese rhesus
   macaque genome and identification of ape-specific structural
   variants. 

  * "We present a high-quality de novo genome assembly (rheMacS) of
    the Chinese rhesus macaque (Macaca mulatta) using long-read
    sequencing and multiplatform scaffolding approaches. Compared to
    the current Indian rhesus macaque reference genome (rheMac8),
    rheMacS increases sequence contiguity 75-fold, closing 21,940 of
    the remaining assembly gaps (60.8 Mbp). We improve gene annotation
    by generating more than two million full-length transcripts from
    ten different tissues by long-read RNA sequencing. We sequence
    resolve 53,916 structural variants (96% novel) and identify 17,000
    ape-specific structural variants (ASSVs) based on comparison to
    ape genomes." 

1. [Immel et al. 2019.][immel19:biorxiv] Neolithic genomes reveal a
   distinct ancient HLA allele pool and population transformation in
   Europe.

    * Studied 42 late-Neolithic genomes from the Wartenberg culture of
      Europe. A high percentage--40%--of their ancestry was from hunter
      gatherers, and their HLA alleles were unlike those of modern
      Europeans. The authors argue that their immune systems were
      specialized for defense against viruses.

1. [Zen, Huber, Davies, and Lohmuller, 2020][zen20:biorxiv] Greater
   strength of selection and higher proportion of beneficial amino
   acid changing mutations in humans compared to mice and Drosophila
   melanogaster.

    * "We use improved modeling of weakly deleterious mutations and the
      demographic history of the outgroup species and ancestral
      population and estimate that at least 20% of nonsynonymous
      substitutions between humans and an outgroup species were fixed by
      positive selection. This estimate is much higher than previous
      estimates, which did not correct for the sizes of the outgroup
      species and ancestral population."

1. [Byrne et al. 2020][byrne20:ng] Dutch population structure across
  space, time and GWAS design.

  * The authors use haplotype sharing to estimate the fine-grained
    population structure and history of the Dutch population. "Despite
    superexponential population growth, regional demographic estimates
    reveal population crashes contemporaneous with the Black
    Death. Within Dutch and international data, GWAS incorporating
    fine-grained haplotypic covariates are less confounded than
    standard methods."

1. [Parreira et al. 2020][parreira20:h] Genetic consequences of social
  structure in the golden-crowned sifaka.

  * "We observe high excess of heterozygotes within
    [social groups (SGs)]. ... While offspring are characterised by a
    high excess of heterozygotes (FIS < 0), the reproductive pair does
    not show dramatic departures from Hardy–Weinberg
    expectations. Moreover, the heterozygosity excess disappears at
    larger geographic scales (sites and forests). ... Our results
    suggest that social structure leads to high levels of outbreeding
    without the need for active inbreeding avoidance mechanisms."

1. [Zeberg and Paabo. 2020][zeberg20:biorxiv] The major genetic risk
   factor for severe COVID-19 is inherited from Neandertals.

    * "A recent genetic association study (Ellinghaus et al. 2020)
       identified a gene cluster on chromosome 3 as a risk locus for
       respiratory failure in SARS-CoV-2. Recent data comprising 3,199
       hospitalized COVID-19 patients and controls reproduce this and
       find that it is the major genetic risk factor for severe
       SARS-CoV-2 infection and hospitalization (COVID-19 Host Genetics
       Initiative). Here, we show that the risk is conferred by a genomic
       segment of ~50 kb that is inherited from Neandertals and occurs at
       a frequency of ~30% in south Asia and ~8% in Europe."

1. [Zeberg, et al. 2020.][zeberg20:mbe] The Neandertal Progesterone
  Receptor.

    * "The gene encoding the progesterone receptor (PGR) carries
      introgressed Neandertal haplotypes...[, which] have reached nearly
      20% frequency in non-Africans and have been associated with
      preterm birth...  Present-day carriers of the Neandertal
      haplotypes express higher levels of the receptor. In a cohort of
      present-day Britons, these carriers have more siblings, fewer
      miscarriages, and less bleeding during early pregnancy suggesting
      that the Neandertal progesterone receptor alleles promote
      fertility. This may explain their high frequency in modern human
      populations."

1. [Hernandez, Shenk, and Perry. 2020][hernandez20:biorxiv] Factors
   influencing taxonomic unevenness in scientific research: A
   mixed-methods case study of non-human primate genomic sequence data
   generation.

    * How do scientists decide which species to focus on in
      DNA-sequencing studies? This study shows that the most-studied
      species are those for which there is a large prior literature and
      that there is a bias in favor of species closely-related to our
      own.

1. [Hicks et al. 2018.][hicks20:nc] Gut microbiomes of wild great apes
   fluctuate seasonally in response to diet.

    * "Here we use 16S rRNA sequencing to characterize the microbiota of
      wild western lowland gorillas and sympatric central chimpanzees
      and demonstrate compositional divergence between the microbiotas
      of gorillas, chimpanzees, Old World monkeys, and modern humans. We
      show that gorilla and chimpanzee microbiomes fluctuate with
      seasonal rainfall patterns and frugivory."

1. [Fernandes et al. 2020][fernandes20:nee] The spread of steppe and
   Iranian-related ancestry in the islands of the western
   Mediterranean.

  * "We generated genome-wide ancient-DNA data from the Balearic
   Islands, Sicily and Sardinia, increasing the number of individuals
   with reported data from 5 to 66. The oldest individual from the
   Balearic Islands (~2400 bc) carried ancestry from steppe
   pastoralists that probably derived from west-to-east migration from
   Iberia, although two later Balearic individuals had less ancestry
   from steppe pastoralists. In Sicily, steppe pastoralist ancestry
   arrived by ~2200 bc. ... Major immigration into Sardinia began in
   the first millennium bc and, at present, no more than 56–62% of
   Sardinian ancestry is from its first farmers. This value is lower
   than previous estimates, highlighting that Sardinia, similar to
   every other region in Europe, has been a stage for major movement
   and mixtures of people." 

1. [Sinding et al. 2020.][sinding20:s] Arctic-adapted dogs emerged at
   the Pleistocene-Holocene transition.

    * "This analysis indicates that sled dogs represent an ancient
      lineage going back at least 9500 years and that wolves bred with
      the ancestors of sled dogs and precontact American dogs. However,
      gene flow between sled dogs and wolves likely stopped before ∼9500
      years ago."

[betti20:n]:
https://www.nature.com/articles/s41562-020-0897-7

[zeberg20:biorxiv]:
https://www.biorxiv.org/content/10.1101/2020.07.03.186296v1

[immel19:biorxiv]:
https://www.biorxiv.org/content/10.1101/851188v1

[zen20:biorxiv]:
https://www.biorxiv.org/content/10.1101/427583v2?%3Fcollection=

[zeberg20:mbe]:
https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msaa119/5841671

[hernandez20:biorxiv]:
https://www.biorxiv.org/content/10.1101/2020.04.16.045450v1

[hicks20:nc]:
https://www.nature.com/articles/s41467-018-04204-w

[sinding20:s]:
https://science.sciencemag.org/content/368/6498/1495.abstract

[mafessoni20:biorxiv]:
https://www.biorxiv.org/content/10.1101/2020.03.12.988956v1

[byrne20:ng]:
https://www.nature.com/articles/s41467-020-18418-4

[he19:nc]:
https://www.nature.com/articles/s41467-019-12174-w

[vanderValk19:mbe]:
https://academic.oup.com/mbe/article/37/1/183/5570178

[parreira20:h]:
https://www.nature.com/articles/s41437-020-0345-5

[fernandes20:nee]:
https://www.nature.com/articles/s41559-020-1102-0

[brandt18:g3]:
https://www.g3journal.org/content/8/8/2805.short

[saag21:sa]:
https://advances.sciencemag.org/content/7/4/eabd6535?fbclid=IwAR1_AkLD_XlR2_nUOQ7L70SAaj7vnCB-bl0Dw31yZB6GS8G31WoEiAY_k3w

[orkin20:biorxiv]:
https://www.biorxiv.org/content/10.1101/366112v6.full

[andirko21:biorxiv]:
https://www.biorxiv.org/content/10.1101/2021.01.22.427608v1

[vizzari20:genes]:
https://www.mdpi.com/2073-4425/11/12/1510

[dominguezandres21:biorxiv]:
https://www.biorxiv.org/content/10.1101/2021.01.14.426690v1

[immel21:commbiol]:
https://www.nature.com/articles/s42003-020-01627-4?fbclid=IwAR3NKeZG6sC7XzQkZ1DxlbyjGxOOzQu7JOBuztxZAAY0SV83Fxg0vtiFsWE

